snap/snapcraft.yaml

```diff
@@ -35,14 +35,17 @@ parts:
       cmake ../src \
         -DCMAKE_INSTALL_PREFIX= \
         -DD_COMPILER=../../ldc-bootstrap/build/bin/ldmd2 \
+        -DD_COMPILER_FLAGS='-flto=full -defaultlib=phobos2-ldc-lto,druntime-ldc-lto' \
+        -DEXTRA_CXXFLAGS='-flto=full' \
```

Comment:
This seems to be causing the compiler to fail: do I need to enable LTO for `ldc-bootstrap` (e.g. the `BUILD_LTO_LIBS` and `LDC_INSTALL_LTOPLUGIN` vars ... ?